esp/smc/SMCLib/TpWrapper.cpp

```diff
@@ -2193,15 +2193,15 @@ extern TPWRAPPER_API unsigned getContainerWUClusterInfo(CConstWUClusterInfoArray
     return clusters.ordinality();
 }
 
-extern TPWRAPPER_API IConstWUClusterInfo* queryContainerWUClusterInfoByName(const char* clustName, CConstWUClusterInfoArray& clusters)
+extern TPWRAPPER_API IConstWUClusterInfo* queryContainerWUClusterInfoByName(const char* clustName)
```

Comment:
should get a getXXX method as it's returning a link (in this case ownership).

Reply:
@wangkx - this one should be changed before merging, the one below less important.

Please squash after changes.

Reply:
Sorry, I missed that.